### DISCUSSION TITLE
custom idle response

### DISCUSSION THREAD

**Discussion by Amin456789** [LostRuins/koboldcpp#477] [Ideas]
Hello!
im a huge fan of idle response as it gives more immersion in chat and very good for a lazy guy like me to not push the send button all the time in story mode :D 

my only idea is: can we have it custom in the future please? i mean x8 and 15 sec be custom as i'd like to make it x99 and 1-2 seconds sometimes.

kinds regards

**Comment by LostRuins** [comment]
I will add a 10x option. My original concern with making it fully customizable is that this UI is also used for AI Horde - and we don't want people to setup idle responses for automatic spam on the Horde. This is fine for local use, but it will be a bit troublesome to customize different options for local usage

**Comment by Amin456789** [comment]
that will be really cool, thank u

**Comment by Amin456789** [comment]
i was thinking, is it possible to code the horde users being triggered to not use over x8 and only local users have the custom option? it can be a solution i guess
